srcalc/src/main/java/gov/va/med/srcalc/domain/model/RiskModel.java

```diff
@@ -17,6 +18,7 @@
  * A risk model definition.
  */
 @Entity
+@Table(name = "risk_model")
```

Comment:
Ok. . This was part of my trial-n-error attempts to fix the bug in the getAllRiskModels() query.